common/components/TXMetaDataPanel/components/SimpleGas.scss

```diff
@@ -9,6 +9,11 @@
     > .SimpleGas-slider {
       flex-grow: 1;
       margin-right: $space;
+
+      > .rc-slider > .rc-slider-step > .rc-slider-dot {
```

Comment:
I'd like the notches to be more noticeable, and they should have a title ideally.. idk how rc-slider handles this.